screwdriver/screwdriver_test.go

```diff
@@ -63,6 +63,10 @@ func TestNew(t *testing.T) {
 		testToken := "token"
 		_, err := New(server.URL, testToken)
 		assert.NotNil(t, err)
+
+		testMsg := "failed to parse JWT response:"
+		assert.Contains(t, err.Error(), testMsg)
```

Comment:
Are the order of the arguments reversed?
```suggestion
		assert.Contains(t, testMsg, err.Error())
```

Reply:
Arguments of assert.Contains are `(t, contains, message)`.
https://godoc.org/github.com/stretchr/testify/assert#Contains

Reply:
Sorry, I see